### PR TITLE
[SD-260] arrows possitions on homepage carousels

### PR DESCRIPTION
**Pull request by pmilewski**

### CODE REVIEW

File: frontend/app/assets/stylesheets/spree/frontend/components-bootstrap/carousel.scss [hunk 1]
```diff
@@ -39,9 +39,7 @@
         top: 0;
         width: 15%;
         z-index: 1;
-        @include media-breakpoint-up(xl) {
-          width: auto;
-        }
+        height: 81% !important;
```
Comment (piotrleniec-spark) [inline frontend/app/assets/stylesheets/spree/frontend/components-bootstrap/carousel.scss]: Is `!important` necessary?

Comment (piotrleniec-spark) [inline frontend/app/assets/stylesheets/spree/frontend/components-bootstrap/carousel.scss]: Are we sure that height set to 81% is responsive across all devices?

**Review by pl-spark — CHANGES_REQUESTED** [review]

**Review by AleksandraKaminska — DISMISSED** [review]
Zrzut ekranu 2019-12-31 o 10 31 25
I think it shouldn't look like this

File: frontend/app/views/spree/home/_carousel.html.erb [hunk 2]
```diff
@@ -6,92 +6,103 @@
           <div class="row">
             <% sliced_items.each do |item| %>
               <div class="col-6">
+                <% if item == sliced_items.first %>
```
Comment (piotrleniec-spark) [inline frontend/app/views/spree/home/_carousel.html.erb]: It'd be cleaner to use `each_with_index` and compare indices instead of using object equality.

**Review by pl-spark — CHANGES_REQUESTED** [review]

**Review by pl-spark — APPROVED** [review]

**Pull request merged**